legal-api/migrations/versions/83a110e10979_add_batch_processing_id_to_colin_event_.py

```diff
@@ -0,0 +1,28 @@
+"""add_batch_processing_id_to_colin_event_ids
+
+Revision ID: 83a110e10979
+Revises: d55dfc5c1358
+Create Date: 2024-10-07 15:34:55.319366
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '83a110e10979'
+down_revision = 'd55dfc5c1358'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('colin_event_ids', sa.Column('batch_processing_id', sa.Integer(), nullable=True))
+    op.add_column('colin_event_ids', sa.Column('batch_processing_step', sa.String(length=1000), nullable=True))
```

Comment:
i think this should use the existing enum to ensure that `batch_processing` uses so we lock down the values that can be used for this column

Reply:
Fixed. please kindly review.